getting_started/basic_walkthrough.adoc

```diff
@@ -122,14 +129,14 @@ image::gs-copy-webhook.png[Copy webhook]
 
 - In the {product-title}  web console:
 .. Navigate to the project containing your application.
-.. Click the *[ Browse ]* tab, then click *[ Builds ]*, then click the name of the
+.. Click the *Browse* tab, then click *Builds*, then click the name of the
 build for your Node.js application.
-.. From the *[ Configuration ]* tab, click image:copy.jpg[] next to *GitHub webhook
+.. From the *Configuration* tab, click image:copy.jpg[] next to *GitHub webhook
```

Comment:
Please specify that the "Content Type" has to be "application/json", as opposed to the default "application/x-www-form-urlencoded"

![screen shot 2017-10-17 at 15 28 52](https://user-images.githubusercontent.com/22222756/31685116-eb05c972-b34f-11e7-83a0-857956801bc2.png)